---
abstract: Data resulting from chromatin immunoprecipitation followed by massively parallel sequencing (ChIP-seq) assays have been used to identify genomic locations where a target protein is bound to DNA. Of interest is the detection of changes in local binding activity across various conditions such as cell lines or treatments. The identification of such differential binding patterns elucidate epigenomic drivers behind conditions and aid the understanding of potential biological processes that lead to a downstream phenotypic impact. Current methods that focus on the detection of differential sites are either restricted to the scenario of two conditions or tailored to narrow enrichment profiles. We present a framework based on a hidden Markov model with embedded mixtures as emission distributions. The model setup generalizes current methods as it allows for the detection of broad binding profiles from numerous conditions. The embedded mixture model permits the detection and classification of the existing binding patterns in the data. We show that our approach outperforms against competing methods on multiple ChIP-seq datasets from the ENCODE and Roadmap projects as well as in simulation.
address:
  city: Denver
  country: United States
  region: CO
all_day: false
authors:
- Pedro L. Baldoni
- Naim U. Rashid
- Joseph G. Ibrahim
date: "2019-07-29"
date_end: "2030-06-01T15:00:00Z"
event: JSM 2019
event_url: https://ww2.amstat.org/meetings/jsm/2019
featured: false
links:
- name: Slides
  url: https://drive.google.com/open?id=1Z85FTkzSQdCk1t6-7S3MoDPOe4_umRG8
location: Colorado Convention Center
math: true
publishDate: "2019-07-29"
slides: 
summary: 
tags: []
title: 	Detection and Classification of Changes in Protein-DNA Binding Activity with Applications in Diffuse ChIP-Seq Data
url_code: ""
url_pdf: ""
url_slides: ""
url_video: ""
---


