---
abstract: Chromatin immunoprecipitation followed by next‐generation sequencing (ChIP‐seq) is a technique to detect genomic regions containing protein‐DNA interaction, such as transcription factor binding sites or regions containing histone modifications. One goal of the analysis of ChIP‐seq experiments is to identify genomic loci enriched for sequencing reads pertaining to DNA bound to the factor of interest. The accurate identification of such regions aids in the understanding of epigenomic marks and gene regulatory mechanisms. Given the reduction of massively parallel sequencing costs, methods to detect consensus regions of enrichment across multiple samples are of interest. Here, we present a statistical model to detect broad consensus regions of enrichment from ChIP‐seq technical or biological replicates through a class of zero‐inflated mixed‐effects hidden Markov models. We show that the proposed model outperforms existing methods for consensus peak calling in common epigenomic marks by accounting for the excess zeros and sample‐specific biases. We apply our method to data from the Encyclopedia of DNA Elements and Roadmap Epigenomics projects and also from an extensive simulation study.
authors:
- Pedro L. Baldoni
- Naim U. Rashid
- Joseph G. Ibrahim
date: "2019-05-13"
doi: "10.1111/biom.13083"
featured: true
projects: []
publication: '*Biometrics*'
publication_short: ""
publication_types:
- "2"
publishDate: "2019-05-13"
title: Improved detection of epigenomic marks with mixed‐effects hidden Markov models
url_code: ""
url_dataset: ""
url_pdf: https://onlinelibrary.wiley.com/doi/abs/10.1111/biom.13083
url_poster: ""
url_project: ""
url_slides: ""
url_source: ""
url_video: ""
---