---
abstract: Chromatin immunoprecipitation followed by next generation sequencing (ChIP-seq) is an experimental technique to detect regions of protein-DNA interaction in the genome, such a transcription factor binding sites or regions containing histone modifications ("enrichment regions"). A common goal of the analysis of ChIP-seq data is to identify genomic loci enriched for sequencing reads pertaining to DNA bound to the factor of interest. Given the reduction of massive parallel sequencing costs over time, novel methods to detect consensus regions of enrichment across multiple samples or differential enrichment between groups of samples are of interest. We develop a statistical method and software to simultaneously analyze multiple histone modification ChIP-seq datasets through a class of efficient Mixed Hidden Markov Models (MHMM) to call consensus broad regions region of enrichment found across samples. This novel methodology will provide a tool able to compare data from different experiments in a single framework by considering subject and population-specific variation in a single model. Simulation studies and real data results will be presented.
address:
  city: Baltimore
  country: United States
  region: MD
all_day: false
authors:
- Pedro L. Baldoni
date: "2017-08-02T11:35:00Z"
date_end: "2017-08-02T12:20:00Z"
event: JSM 2017
event_url: https://ww2.amstat.org/meetings/jsm/2017/
featured: false
location: JSM 2017
math: true
publishDate: "2019-11-1T10:35:00Z"
slides: 
summary: 
tags: []
title: A Statistical Method for the Analysis of Multiple ChIP-Seq Data Sets
url_code: ""
url_pdf: ""
url_slides: ""
url_video: ""
---


